src/test/resources/csv/all_rand_normal_with_deleted.csv

```diff
@@ -0,0 +1,38 @@
+x1,x2,x3,x4,x5,x6,x7,x8
+0,1.77283196480627,1.55961845104152,-0.0754841337123781,-0.127310386905666,1.27160971116606,-0.373485088927311,-0.491478299933984
+1,0.0196004307029624,-0.413369625750338,-0.168943253776815,0.548823278255197,0.190611907286572,0.461311082111462,-0.183821951945449
+2,1.71152031127659,-0.343253067969038,-2.4441579734022,0.771278181748259,-0.790959408495155,-1.1678181616338,-1.42230765453915
+3,0.283548642994546,0.506250013338169,0.73547312521291,1.0921602561092,-1.42634104687771,-0.547784365367733,-0.0679134396910904
+4,-0.0283595950513087,1.19265624530262,-1.95700253011149,-1.31936124633078,1.13583204842472,-0.773892287552427,0.0306183416824843
+5,-1.22009820639777,-0.853206793552464,-1.74368068722334,2.59744290896532,0.283189394665112,1.06409063075838,-1.07707365923793
+6,-1.30601106460683,-0.191645716555782,-0.890895281080635,-1.01420554584294,0.239205026896137,-1.36956281497245,0.367116358963696
+7,-0.182529555280925,1.70588149747924,-0.648252455053375,-0.113856279856997,-1.59414475661726,-0.0716625039715658,0.422675801034446
+8,1.55587487228279,-0.240535685327487,-0.987747968432627,0.379810225477961,-0.616028300364221,-1.87098495560613,0.63376130380024
+9,0.747240400228516,0.0760947073326997,-0.755460377522965,0.146127904800772,-1.22177835494399,-0.709271266564636,2.57184269865151
+10,-0.279883956321368,1.47435132507343,-0.948816009788528,-0.844456583968411,0.139119584115725,1.35427048218451,-1.27013451653836
+20,1.41199698228818,0.836223566992313,0.266081420325081,-1.07577894523886,-0.776099337225336,-0.292190865045215,-0.629139226286168
+30,-1.25875867828822,0.185198950004779,-0.116324903204208,-1.03625723019971,-0.0170051390322138,-0.322461408145671,0.00632738879553362
+40,0.891389660634871,-0.883007989589512,-0.654361699302352,1.26265202786226,1.13288576340466,-0.0927398141356724,0.0672848024456528
+50,2.15000039664857,0.749410953295052,-1.16764981806199,0.466677499486481,-0.0552601977313928,0.0143111301094924,-0.39946607189219
```

Comment:
@Yana-Guseva So this record in the SAS version was removed. The way SAS works in some cases is that is does not purge from the file in non compressed versions. We need to inspect the Subheaders and find the pointers that say "this record is invalid". I believe it is hidden somewhere in the Row Size Subheader.

Reply:
to make it clear, the record with the value '50' in the first column